Library/Homebrew/cask/cask_loader.rb

```diff
@@ -256,7 +263,7 @@ def load(config:)
           desc json_cask[:desc]
           homepage json_cask[:homepage]
 
-          auto_updates json_cask[:auto_updates] if json_cask[:auto_updates].present?
+          auto_updates json_cask[:auto_updates] unless json_cask[:auto_updates].nil?
```

Comment:
There was at least one cask that had `auto_updates: false` and so this fixes that case. I don't think there is a functional difference though.